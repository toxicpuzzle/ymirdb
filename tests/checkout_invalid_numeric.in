CHECKOUT a1bc
CHECKOUT zzz
bye