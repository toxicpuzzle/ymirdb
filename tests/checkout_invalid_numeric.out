> You must provide a valid ID for a snapshot!

> You must provide a valid ID for a snapshot!

> bye